src/ert/_c_wrappers/enkf/ensemble_config.py

```diff
@@ -286,6 +286,10 @@ def get_surface_node(surface: List[str]) -> EnkfConfigNode:
     def get_field_node(
         field: Union[dict, list], grid: Optional[EclGrid]
     ) -> EnkfConfigNode:
+        if grid is None:
```

Comment:
If `grid` shouldn't be `None`, I guess the signature of this function should change to:

```
    def get_field_node( field: Union[dict, list], grid: EclGrid) -> EnkfConfigNode:
```

Also while you are making changes in this file can you fix the GitHub action pylint suggestion from ln473

Reply:
Fixed.